src/data/polynomial/eval.lean

```diff
@@ -222,7 +222,13 @@ lemma eval_sum (p : polynomial R) (f : ℕ → R → polynomial R) (x : R) :
   (p.sum f).eval x = p.sum (λ n a, (f n a).eval x) :=
 eval₂_sum _ _ _ _
 
-
+lemma eval_finset_sum (s : finset ι) (g : ι → polynomial R) (x : R) :
```

Comment:
Is this from #4286 ? If so, you might want to mark this PR as depending on that one.

Reply:
I have added a "block-by-other-pr" label.